<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointAnalysisClass</class>
 <widget class="QMainWindow" name="PointAnalysisClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PointAnalysis</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyGLWidget" name="displayGLWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="mainProgressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>740</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mainProgressMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="debugTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="symPlaneCheckBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>570</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw symmetry planes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="symAxisCheckBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>590</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw symmetry axes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="angleInput">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>690</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;90&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>700</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>740</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Axis</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="xAxisInput">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>730</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="yAxisInput">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>730</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="zAxisInput">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>730</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>740</y>
      <width>10</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>740</y>
      <width>10</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>730</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="drawOBBCheckBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>630</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw oritend bounding boxes</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="drawOBBAxesCheckBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>660</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw axes of OBBs</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_Vertices_Labels"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFeature">
    <property name="title">
     <string>Train</string>
    </property>
    <addaction name="actionEstimate_Features"/>
    <addaction name="actionExtract_Point_Features"/>
    <addaction name="actionTrain_Point_Classifier"/>
    <addaction name="actionCompute_sdf"/>
    <addaction name="actionNormalize_Meshes"/>
    <addaction name="actionCompute_OBB"/>
    <addaction name="actionTrain_Parts_Relations"/>
    <addaction name="actionDebug_Parts_Relations"/>
    <addaction name="actionCheck_Models"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionTest_Data"/>
    <addaction name="actionTest_PointCloud"/>
   </widget>
   <widget class="QMenu" name="menuInference">
    <property name="title">
     <string>Inference</string>
    </property>
    <addaction name="actionStructure_Inference"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionDownsample"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFeature"/>
   <addaction name="menuTest"/>
   <addaction name="menuInference"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionEstimate_Features">
   <property name="text">
    <string>Estimate Features</string>
   </property>
  </action>
  <action name="actionExtract_Point_Features">
   <property name="text">
    <string>Extract Point Features</string>
   </property>
  </action>
  <action name="actionTrain_Point_Classifier">
   <property name="text">
    <string>Train Point Classifier</string>
   </property>
  </action>
  <action name="actionTest_Data">
   <property name="text">
    <string>Test Data</string>
   </property>
  </action>
  <action name="actionTest_PointCloud">
   <property name="text">
    <string>Test Point Cloud</string>
   </property>
  </action>
  <action name="actionCompute_sdf">
   <property name="text">
    <string>Compute sdf</string>
   </property>
  </action>
  <action name="actionNormalize_Meshes">
   <property name="text">
    <string>Normalize Meshes</string>
   </property>
  </action>
  <action name="actionCompute_OBB">
   <property name="text">
    <string>Compute OBB</string>
   </property>
  </action>
  <action name="actionTrain_Parts_Relations">
   <property name="text">
    <string>Train Parts Relations</string>
   </property>
  </action>
  <action name="actionStructure_Inference">
   <property name="text">
    <string>Structure Inference</string>
   </property>
  </action>
  <action name="actionDebug_Parts_Relations">
   <property name="text">
    <string>Debug Parts Relations</string>
   </property>
  </action>
  <action name="actionCheck_Models">
   <property name="text">
    <string>Check Models</string>
   </property>
  </action>
  <action name="actionSave_Vertices_Labels">
   <property name="text">
    <string>Save Vertices Labels</string>
   </property>
  </action>
  <action name="actionDownsample">
   <property name="text">
    <string>Downsample</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">myglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pointanalysis.qrc"/>
 </resources>
 <connections/>
</ui>
